features/logging/cluster_logging_operator.feature

```diff
@@ -126,13 +126,13 @@ Feature: cluster-logging-operator related test
     """
     And the expression should be true> elasticsearch('elasticsearch').policy_ref(name: 'app') == "app-policy"
     And the expression should be true> elasticsearch('elasticsearch').delete_min_age(name: "app-policy") == cluster_logging('instance').application_max_age
-    And the expression should be true> elasticsearch('elasticsearch').rollover_max_age(name: "app-policy") == "1h"
+    And the expression should be true> elasticsearch('elasticsearch').rollover_max_age(name: "app-policy") == "60m"
```

Comment:
this value should be the `maxAge` in elasticsearch/elasticsearch. So here the values are `3m`,  `9m`, `1h`